deployment/arc-k8s-extension/ServiceGroupRoot/Scripts/pushChartToAcr.sh

```diff
@@ -133,64 +118,64 @@ if [ $? -eq 0 ]; then
 else
   echo "-e error login to acr:${ACR_NAME} using helm failed. Please review Ev2 pipeline logs for more details on the error."
   exit 1
-fi   
+fi
 
 case $RELEASE_STAGE in
 
   Canary)
     echo "START: Release stage - Canary"
-    destAcrFullPath=${ACR_NAME}/public/${CANARY_REGION_REPO_PATH}:${CHART_VERSION}  
-    push_local_chart_to_canary_region $destAcrFullPath     
+    destAcrFullPath=oci://${ACR_NAME}/public/${CANARY_REGION_REPO_PATH}
```

Comment:
What is oci?

Reply:
> What is oci?
OCI stands for Open Container Initiative. This open standard for helm chart package format.
some docs related to this. 
https://learn.microsoft.com/en-us/azure/container-registry/container-registry-helm-repos
https://helm.sh/blog/storing-charts-in-oci/